build.gradle

```diff
@@ -85,3 +86,13 @@ spotless {
         ktlint(ktlintVersion)
     }
 }
+
+subprojects {
```

Comment:
how come this is needed now?

Reply:
This is the first time the `@OptIn` annotation is needed